### PR TITLE
Remove code that was masking errors and panics by running

### LINKED ISSUE #286: main.go lines 13 to 16 masks panics by exiting without errors
## Bug Description
opcap fails on audit.go line 59 with out of boundary message.
opcap just exits with no errors

## Version and Command Invocation
opcap check

## Steps to Reproduce:
1) just run opcap check

## Expected Result
Run audits against operators

## Actual Result
The program exits without errors and doesn't do it's job

## Additional Context
No additional context

### PR DESCRIPTION
**Pull request by acmenezes**
Signed-off-by: acmenezes <adcmenezes@gmail.com>

## Description of PR
Exit function with code 0 was masking errors and panics.

Fixes #286 

## Changes (required)
- Remove exitCode and related function from main.go

## Checklist (required)
- [x] I have reviewed and followed the [contribution guidelines](https://github.com/opdev/opcap/blob/main/docs/contribution.md)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation if needed
- [x] I have checked that my changes pass all tests

## References (optional)

### CODE REVIEW

**Review by yashoza19 — APPROVED** [review]

**Comment by mrhillsman** [comment]
@yashoza19 please lgtm if you are performing initial review rather than approving. additionally have the checks that failed been reviewed or are they non-consequential?

**Review by bcrochet — APPROVED** [review]
/lgtm 

We may want to revisit this later, but probably a good idea to not mask anything for the moment.

File: main.go [hunk 1]
```diff
@@ -10,18 +10,13 @@ import (
 )
 
 func main() {
-	exitCode := 0
-	defer func() {
-		os.Exit(exitCode)
-	}()
```
Comment (bcrochet) [inline main.go:13]: You'll need to remove this empty line.

**Comment by bcrochet** [comment]
TBH, 

> @yashoza19 please lgtm if you are performing initial review rather than approving. additionally have the checks that failed been reviewed or are they non-consequential?

Empty line definitely should be removed before merging.

File: main.go (hunk 1 above)
Comment (acmenezes) [reply main.go]: Not sure why but line removed.

Comment (bcrochet) [reply main.go]: Part of gofumpt. No empty lines after function declaration.

Comment (bcrochet) [reply main.go]:
```
No empty lines around function bodies
example

func foo() {

	println("bar")

}

func foo() {
	println("bar")
}
```

Comment (acmenezes) [reply main.go]: Oh got it. It's removed anyway. I guess it's good now. Let me know if there is anything else that I should do here.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**